---
name: Gibran Hemani
position: "Associate professor of statistical genetics, University of Bristol"
phone: "+44(0)7930951876"
www: gibranhemani.com
email: g.hemani@bristol.ac.uk
github: explodecomputer
aboutme:
date: "`r format(Sys.time(), '%B %Y')`"
headcolor: "000000"
output:
  vitae::awesomecv:
    keep_tex: true
csl: "../resources/nature.csl"
---

```{r setup, include=FALSE}
knitr::opts_chunk$set(echo = FALSE, 
                      warning = FALSE,
                      message = FALSE)
library(tidyverse)
library(vitae)
library(bib2df)
library(here)
library(glue)

separate_why <- function(x)
{
  x %>%
    dplyr::mutate(why = strsplit(why, "\\. ")) %>%
    tidyr::unnest(why)
}
```

# Present Appointment

```{r employment}
jobs <- read_csv(here("data", "jobs.csv")) 

jobs[1,] %>%
  detailed_entries(
    what = role,
    when = glue("{start_date} to {end_date}"),
    with = company,
    where = where,
    why = details)
```

# Previous Appointments

```{r}
jobs[-1,] %>%
  detailed_entries(
    what = role,
    when = glue("{start_date} to {end_date}"),
    with = company,
    where = where,
    why = details)
```

# Academic Qualifications

```{r}
read_csv(here("data", "education.csv")) %>%
  detailed_entries(
    when = glue("{start_date} to {end_date}"),
    what = degree,
    where = loc,
    with = uni,
    why = details
)
```

# Special Awards, Honours and Distinctions

```{r awards}
read_csv(here("data", "awards.csv")) %>%
  arrange(desc(when)) %>%
  detailed_entries(
    with = what,
    what = why,
    when = when,
    where = where
)
```

\newpage

# Teaching and related administration

The teaching-research nexus has been central to my career, starting during my PhD when I wrote and delivered a 2-day course on R programming for researchers at the Roslin Institute. Since then I have taught on topics relating to genetics, statistics and programming to undergraduates across disciplines in medicine, biology and psychology. There has been a major shift towards using genetic data in epidemiological research over the last several years and my academic background in statistical genetics has created opportunities and substantial demand for teaching in this field at the post-graduate and life-long learning levels. I am particularly proud of my record of writing and delivering courses on genetic epidemiology to participants who are under-represented in genomic data science. In line with this I am also co-leading the BMS Decolonising the Curriculum group, which is now piloting a curriculum review framework that we developed for Bristol Medical School.

## (i) Undergraduate and taught postgraduate (past 3 years)

```{r}

## Teaching
teaching <- read_csv(here::here("data", "teaching.csv")) %>% 
  arrange(desc(when)) %>%
  separate_why()

teaching %>% 
  subset(., type == "taught ug and pg") %>%
  detailed_entries(with=with, what = what, when = when, why = why, where=where)
```

## (ii) Major teaching responsibilities in previous years

```{r}
teaching %>% 
  subset(., type == "previous teaching") %>%
  detailed_entries(with=with, what = what, when = when, why = why, where=where)
```

\newpage

## (iii) Innovatory units and teaching methods

**Flipped classroom:** In the second year of the iBSc Genomic Medicine course I redesigned the R programming training section to be a flipped classroom, using online games for students to learn the basic principles of programming at home and the tutor led sessions as an opportunity to synthesise those skills into applied examples. The students were more adept at programming than in the previous year, and also more enthused about the subject. I transitioned the rest of the module to a flipped classroom in 2020 during the Covid-19 pandemic.

## (iv) Contribution to Life-long learning and continuing professional development courses

```{r}
teaching %>% 
  subset(., type %in% c("shortcourse", "lecturing contributions")) %>%
  detailed_entries(with=with, what = what, when = when, why = why, where=where)
```

## (v) Collaborative teaching projects

Developed external speaker programmes for iBSc medical students, involving Jeff Barrett from OpenTargets and the Sanger Institute; Rob Scott from GlaxoSmithKlein; and Jonathan Ives from the Centre for Ethics in Medicine.

\newpage

## (iv) Postgraduate advising

```{r}
postgrad <- read_csv(here::here("data", "postgrad.csv")) %>% 
  arrange(desc(when)) %>%
  separate_why()

postgrad %>% 
  subset(., where == "phd") %>%
  detailed_entries(what = with, when = when, with = what, why = why)
```

## (vii) Major achievements in teaching administration

**International short courses on statistical genetics:** I have organised, developed and delivered short courses (typically week-long, in collaboration with internal and/or external colleagues) to researchers at institutes in South Africa, Italy, Brazil and at the Wellcome Genome Centre in Cambridge.

**Genomic Data Science module:** I designed, organised and delivered a 20 credit module for medical students that involves teaching them programming and genomic data analysis from scratch. This has run for 4 years, and consistently gets very positive feedback, and the students achieve a high standard.

**Decolonising the curriculum:** I lead the Medical Anti-Racism Taskforce Special Interest Group on decolonising the curriculum in the Bristol Medical School (11 members). We are developing a curriculum review framework that is now being piloted in five modules across the school (http://decolbms.org.uk). I also liaise with numerous related groups across the university.

**MR-Base software course:** I developed a course to teach researchers how to use my own software (MR-Base), with the course being heavily over-subscribed and gaining very positive feedback and increased usage of the software.

# Research and related administration

During my PhD I developed statistical and computational methods to detect genetic interactions influencing complex traits. This work led to my first post-doctoral position that expanded my work into determining genetic architectures of complex disease, and developing expertise in handling high dimensional genomic data. My fellowship work lies at the interface between statistical genetics, causal inference and high performance computing. A major output that integrates these fields has been the development of the MR-Base and OpenGWAS causal inference analytical platform. These software platforms are used by researchers globally and has led directly to funding for five post-doctoral positions from three private companies (GSK, Biogen, CHDI). I currently lead or co-lead three major international collaborations. These activities will continue and expand in 2023 as co-lead of the Mendelian randomisation programme in MRC IEU 3.0.

## (i) Publications

```{r}
load(here("data", "pubs.rdata"))
convert_to_bib <- function(x)
{
  Encoding(x$author) <- "UTF-8"
  x$author <- iconv(x$author, "UTF-8", "UTF-8", sub="")
  x$pubid <- gsub("-", "", x$pubid)
  x$author <- sapply(x$author, function(string)
  {
    authors = trimws(unlist(strsplit(string, ",")))
    format_author = function(author) {
        words = trimws(unlist(strsplit(author, " ")))
        lastname = words[length(words)]
        first = words[!words %in% lastname]
        first = gsub("\\B[a-z]", "", first, perl = TRUE)
        first = paste(first, collapse = "")
        trimws(paste(first, lastname))
    }
    authors = sapply(authors, format_author)
    paste(authors, collapse = ", ")
  })

  tmpfile <- here("data", "pubs.bib")
  tibble(
    CATEGORY="ARTICLE",
    BIBTEXKEY=x$pubid,
    AUTHOR=
      strsplit(x$author, split=", ") %>%
      lapply(., function(i) {
        stringr::str_split(i, " ", n=2) %>%
        sapply(., function(j) {
          paste(rev(j), collapse=", ")
          }) %>%
        gsub("al, et", "others", .)
      }),
    TITLE=x$title,
    JOURNAL=x$journal,
    NUMBER=x$number,
    YEAR=x$year,
    NOTE=gsub("%", "\\\\%", x$info)
  ) %>% bib2df::df2bib(file=tmpfile)
  b <- bibliography_entries(tmpfile)
  a <- gsub("\\\\leavevmode", "\\\\item \\\\leavevmode", readLines(tmpfile))
  writeLines(a, tmpfile)
  return(b)
}
```

Selected publications organised by category are listed below. Full publication list on [google scholar](https://scholar.google.ca/citations?hl=en&user=6fC0BYYAAAAJ).

- H-index: **`r h_index`**
- i10-index: **`r i10_index`**
- Number of publications: **`r nrow(pubs)`**

### Chapters in Edited Books

```{r}
spubs %>% 
  dplyr::filter(type == "book chapter") %>%
  convert_to_bib()
```


### Academic journal papers (refereed)

```{r}
spubs %>% 
  dplyr::filter(type == "peer reviewed journal article") %>%
  convert_to_bib()
```

### Review articles

```{r}
spubs %>% 
  dplyr::filter(type == "review article") %>%
  convert_to_bib()
```

\newpage

### Selected published open source software

```{r}
software <- read_csv(here::here("data", "software.csv")) 
bind_rows(
  software %>% mutate(why=what),
  software %>% mutate(why=where)
) %>% 
  arrange(desc(when), package) %>%
  detailed_entries(what = package, when = when, why = why)
```

\newpage

## (ii) Forthcoming publications

```{r}
spubs %>% 
  dplyr::filter(type == "preprint") %>%
  convert_to_bib()
```

## (iii) Research grants

```{r}
grants <- read_csv(here::here("data", "grants.csv")) %>%
  mutate(start=as.Date(start, format=c("%d/%m/%Y")), end=as.Date(end, format=c("%d/%m/%Y"))) %>%
  filter(awarded != "pending") %>%
  arrange(desc(start))

grant_totals <- grants %>% 
  group_by(role) %>% 
  summarise(n=n(), income=sum(amount * conversion), incomep = format(round(as.numeric(income), 2), nsmall=0, scientific=FALSE, big.mark=","))

grant_totals_current <- grants %>% 
  subset(., end > today()) %>%
  group_by(role) %>% 
  summarise(n=n(), income=sum(amount * conversion), incomep = format(round(as.numeric(income), 2), nsmall=0, scientific=FALSE, big.mark=","))


grantsf <- grants %>% 
  mutate(n=1:nrow(grants)) %>%
  group_by(n) %>%
  do({
    x <- .
    merge(
      x, 
      tibble(n=x$n[1],
        why=paste(
          "Amount: ", paste(format(round(as.numeric(x$amount)), nsmall=0, scientific=FALSE, big.mark=","), x$currency),
          "; Role: ", x$role,
          "; Dates: ", paste(x$start, "to", x$end),
          "; Proportion: ", x$perc),
        when=x$awarded
      )
    )
  })
```

Total income as PI, from `r subset(grant_totals, role=="PI")$n` grant(s): **`r subset(grant_totals, role=="PI")$incomep` GBP**

Total income as CI, from `r subset(grant_totals, role=="Co-I")$n` grant(s): **`r subset(grant_totals, role=="Co-I")$incomep` GBP**

Current income as PI, from `r subset(grant_totals_current, role=="PI")$n` grant(s): **`r subset(grant_totals_current, role=="PI")$incomep` GBP**

Current income as CI, from `r subset(grant_totals_current, role=="Co-I")$n` grant(s): **`r subset(grant_totals_current, role=="Co-I")$incomep` GBP**


```{r}
grantsf  %>%
  detailed_entries(with=title, why=why, when=when, what=funder)
```

## (iv) Indications of external recognition

### Editorships

```{r}
read_csv(here::here("data", "editorships.csv")) %>%
  arrange(desc(when)) %>%
  detailed_entries(what=what, when=when, with=with)
```

### Appointment to national or international bodies

```{r}
read_csv(here::here("data", "bodies.csv")) %>%
  arrange(desc(when)) %>%
  detailed_entries(what=what, when=when, with=with)
```

### Invitations for degree examinations

```{r}
read_csv(here::here("data", "degree_examination.csv")) %>%
  arrange(desc(when)) %>%
  detailed_entries(what=what, where=where, with=who, when=when)
```

\newpage

### Invited lectures (last 3 years)

```{r}
read_csv(here::here("data", "talks.csv")) %>% 
  arrange(desc(when)) %>%
  filter(when >= (now() %>% year() - 3)) %>%
  arrange(desc(when)) %>%
  detailed_entries(what=what, with=who, when=when, where=type, why=details)
```

## (vi) Related administration

```{r}
read_csv(here::here("data", "research_management.csv")) %>%
  mutate(where = as.character(where)) %>%
  arrange(desc(when)) %>%
  detailed_entries(what=with, where=where, with=what, when=when, why=why)
```

# Academic leadership and citizenship

## (i) Academic leadership in the discipline

I orchestrate the collection and harmonisation of summary data from genome-wide association studies (GWAS) across hundreds of international research groups through the OpenGWAS project. GWAS summary data is vitally important to a wide range of genetic and epidemiological research activities, and the OpenGWAS infrastructure is the largest of its kind comprising approximately 125 billion genetic associations. It has more than 1000 unique users a month and receives more than 2 million queries per week from all over the world. 

I design analytical pipelines by unifying and developing statistical methods with researchers from multiple international groups. For example I developed the MR-Base software platform, which has now been cited over 2200 times since being published in 2018. 

I co-lead three international collaborative research efforts: the GoDMC consortium comprising 56 cohorts analysing the genetics of DNA methylation; the OpenGWAS consortium that combines the resources of over 100 groups and consortia; and the Within-Families consortium which brings together 21 family cohorts with genetic and phenotypic data for improved genetic association meta-analyses

I contributed to the design of the Our Future Health Project, which will genotype 5 million individuals in the UK, as a member of the chip design committee and as an expert reviewer in the procurement stage of the project.

```{r}
academic_leadership <- read_csv(here::here("data", "academic_leadership.csv")) %>%
  mutate(where = as.character(where), why = as.character(why)) %>%
  arrange(desc(when)) %>%
  separate_why()

# academic_leadership %>%
#   filter(type == "international") %>%
#   arrange(desc(when)) %>%
#   detailed_entries(what=as.character(what), where=where, with=with, when=when, why=why)
```

\newpage

## (ii) Academic leadership in the University

```{r}
academic_leadership %>%
  filter(type == "university") %>%
  arrange(desc(when)) %>%
  detailed_entries(what=as.character(what), where=where, with=with, when=when, why=why)
```

## (iii) Professional activities outside the University

```{r}
academic_misc <- read_csv(here::here("data", "academic_misc.csv")) %>%
  arrange(desc(when))

academic_misc %>%
  filter(type=="professional") %>%
  brief_entries(what=what, when=when)
```

\newpage

## (iv) Contributions to society

```{r}
academic_misc %>%
  filter(type=="society") %>%
  brief_entries(what=what, when=when)
```

## (v) Entrepreneurship, enterprise and partnerships

```{r}
academic_misc %>%
  filter(type=="entrepreneurship") %>%
  brief_entries(what=what, when=when)
```

## (vi) Good citizenship

```{r}
academic_misc %>%
  filter(type=="citizenship") %>%
  brief_entries(what=what, when=when)
```

# Future plans

I am committed to grow as an academic by understanding, refining and improving the impact of my work.

**Teaching**: I plan to develop a new professional development course that guides junior researchers and post-graduate students through best practices in code and data management and reproducible digital research. This reflects an important direction for me, of building a teaching-research nexus which I will continue to explore in my decolonisation activities and contributions to life-long learning courses.

**Administration**: Team science is key to how I work, and though I have had a number of opportunities to lead teams thus far, I will seek opportunities to build my skills in this area, in terms of best serving the projects and people involved.

**Research**: I co-developed an extensive research plan for the next 5 years of the IEU relating to method development for causal inference and application to the ever changing landscape of genetic and genomic data. However, the structure of MRC units will change after this period, towards one which appears to be more goal oriented around applied research, and on a much longer time horizon than my work has previously worked towards. I will spend this time building ideas and plans for this next phase. I am interested in integrating my decolonisation work, for example by understanding how public and patient involvement may operate with the new wave of deep learning approaches that are yet to make a large impact in participatory genetic epidemiology.
