---
name: Gibran Hemani
position: "Senior Research Fellow, University of Bristol"
address: "8 Picton Mews, Bristol, BS6 5PB, United Kingdom"
phone: "+44(0)7930951876"
www: explodecomputer.com
email: explodecomputer@gmail.com
twitter: explodecomputer
github: explodecomputer
aboutme:
date: "`r format(Sys.time(), '%B %Y')`"
headcolor: "000000"
output: vitae::awesomecv
csl: "../resources/nature.csl"
---

```{r setup, include=FALSE}
knitr::opts_chunk$set(echo = FALSE, 
                      warning = FALSE,
                      message = FALSE)
source("../scripts/install.R")

separate_why <- function(x)
{
  x %>%
    dplyr::mutate(why = strsplit(why, "\\. ")) %>%
    tidyr::unnest(why)
}
```

# Present Appointment

```{r employment}
jobs <- read_csv(here("data", "jobs.csv")) 

jobs[1,] %>%
  detailed_entries(
    what = role,
    when = glue("{start_date} to {end_date}"),
    with = company,
    where = where,
    why = details)
```

# Previous Appointments

```{r}
jobs[-1,] %>%
  detailed_entries(
    what = role,
    when = glue("{start_date} to {end_date}"),
    with = company,
    where = where,
    why = details)
```

# Academic Qualifications

```{r}
read_csv(here("data", "education.csv")) %>%
  detailed_entries(
    when = glue("{start_date} to {end_date}"),
    what = degree,
    where = loc,
    with = uni,
    why = details
)
```

# Special Awards, Honours and Distinctions

```{r awards}
read_csv(here("data", "awards.csv")) %>%
  detailed_entries(
    with = what,
    what = why,
    when = when,
    where = where
)
```


# Teaching and related administration

Though I have only had research-based academic appointments, I have found many opportunities to develop a variety of different teaching materials from my PhD to present. There has been a major shift towards using genetic data in epidemiological research over the last several years. My academic background in statistical genetics has created opportunities and demand for teaching in this field. I am particularly proud of my record of delivering courses on genetic epidemiology to participants who are under represented in genomic data science. In line with this I am also leading the Medical School's Anti-Racism special interest group in Decolonising the Curriculum.

## (i) Undergraduate and taught postgraduate (past 3 years)

```{r}

## Teaching
teaching <- read_csv(here::here("data", "teaching.csv")) %>% 
  arrange(desc(when)) %>%
  separate_why()

teaching %>% 
  subset(., type == "taught ug and pg") %>%
  detailed_entries(with=with, what = what, when = when, why = why, where=where)
```

## (ii) Major teaching responsibilities in previous years

```{r}
teaching %>% 
  subset(., type == "previous teaching") %>%
  detailed_entries(with=with, what = what, when = when, why = why, where=where)
```

## (iii) Innovatory units and teaching methods

**Flipped classroom:** In the second year of the iBSc Genomic Medicine course I redesigned the R programming training section to be a flipped classroom, using online games for students to learn the basic principles of programming at home and the tutor led sessions as an opportunity to synthesise those skills into applied examples. The students were more adept at programming than in the previous year, and also more enthused about the subject. I transitioned the rest of the module to a flipped classroom in 2020 during the Covid-19 pandemic.

## (iv) Contribution to Life-long learning and continuing professional development courses

```{r}
teaching %>% 
  subset(., type %in% c("shortcourse", "lecturing contributions")) %>%
  detailed_entries(with=with, what = what, when = when, why = why, where=where)
```

## (v) Collaborative teaching projects

Developed external speaker programmes for iBSc medical students, involving Jeff Barrett from OpenTargets and the Sanger Institute; Rob Scott from GlaxoSmithKlein; and Jonathan Ives from the Centre for Ethics in Medicine.

## (iv) Postgraduate advising

#### PhD Supervision

```{r}
postgrad <- read_csv(here::here("data", "postgrad.csv")) %>% 
  arrange(desc(when)) %>%
  separate_why()

postgrad %>% 
  subset(., where == "phd") %>%
  detailed_entries(what = with, when = when, with = what, why = why)
```

## (vii) Major achievements in teaching administration

**International short courses on statistical genetics:** I have organised, developed and delivered short courses (typically week-long, in collaboration with internal and/or external colleagues) to researchers at institutes in South Africa, Italy, Brazil and at the Wellcome Genome Centre in Cambridge.

**Genomic Data Science module:** I designed, organised and delivered a 20 credit module for medical students that involves teaching them programming and genomic data analysis from scratch. This has run for 4 years, and consistently gets very positive feedback, and the students achieve a high standard.

**Anti-racism:** I co-lead the Special Interest Group of 11 group members on decolonising the curriculum in the Bristol Medical School. I am leading the creation of a framework by which course leads across the school can identify racial biases in their teaching methods and materials, work towards addressing them, and obtaining independent course review from external peers. The group meets approximately every two weeks, and I liaise with the University's central decolonising group regularly also.

**MR-Base software course:** I developed a course to teach researchers how to use my own software (MR-Base), with the course being heavily over-subscribed and gaining very positive feedback and increased usage of the software.

# Research and related administration

My research lies at the interface between statistical genetics, causal inference and high performance computing. A major output that integrates these fields has been the development of the MR-Base causal inference analytical platform. This software is used by researchers within the institute and around the world, has led directly to funding for five post-doctoral positions from three private companies (GSK, Biogen, CHDI). I currently lead or co-lead two major international collaborations: The GoDMC consortium comprising 56 cohorts analysing the genetics of DNA methylation; the OpenGWAS consortium that combines the resources of over 100 groups and consortia.


## (i) Publications

```{r}
load(here("data", "pubs.rdata"))
convert_to_bib <- function(x)
{
  Encoding(x$author) <- "UTF-8"
  x$author <- iconv(x$author, "UTF-8", "UTF-8", sub="")
  tmpfile <- here("data", "pubs.bib")
  tibble(
    CATEGORY="ARTICLE",
    BIBTEXKEY=x$pubid,
    AUTHOR=
      strsplit(x$author, split=", ") %>%
      lapply(., function(i) {
        stringr::str_split(i, " ", n=2) %>%
        sapply(., function(j) {
          paste(rev(j), collapse=", ")
          }) %>%
        gsub("al, et", "others", .)
      }),
    TITLE=x$title,
    JOURNAL=x$journal,
    NUMBER=x$number,
    YEAR=x$year,
    NOTE=x$info
  ) %>% bib2df::df2bib(file=tmpfile)
  bibliography_entries(tmpfile)
}
```

Selected publications organised by category are listed below. Full publication list on [google scholar](https://scholar.google.ca/citations?hl=en&user=6fC0BYYAAAAJ).

- H-index: **`r h_index`**
- i10-index: **`r i10_index`**
- Number of publications: **`r npubs`**

**Academic journal papers (refereed)**

```{r}
pubs %>% 
  dplyr::filter(type == "peer reviewed journal article") %>%
  convert_to_bib()
```

**Review articles**

```{r}
pubs %>% 
  dplyr::filter(type == "review article") %>%
  convert_to_bib()
```

**Selected published open source software**

```{r}
software <- read_csv(here::here("data", "software.csv")) 
bind_rows(
  software %>% mutate(why=what),
  software %>% mutate(why=where)
) %>% 
  arrange(desc(when), package) %>%
  detailed_entries(what = package, when = when, why = why)
```


## (ii) Forthcoming publications

```{r}
pubs %>% 
  dplyr::filter(type == "forthcoming") %>%
  convert_to_bib()
```



## (iii) Research grants



```{r}
grants <- read_csv(here::here("data", "grants.csv")) %>%
  mutate(start=as.Date(start, format=c("%d/%m/%Y")), end=as.Date(end, format=c("%d/%m/%Y"))) %>%
  arrange(desc(start))

grant_totals <- grants %>% 
  group_by(role) %>% 
  summarise(n=n(), income=sum(amount * conversion), incomep = format(round(as.numeric(income), 2), nsmall=0, scientific=FALSE, big.mark=","))

grant_totals_current <- grants %>% 
  subset(., end > today()) %>%
  group_by(role) %>% 
  summarise(n=n(), income=sum(amount * conversion), incomep = format(round(as.numeric(income), 2), nsmall=0, scientific=FALSE, big.mark=","))


grantsf <- grants %>% 
  mutate(n=1:nrow(grants)) %>%
  group_by(n) %>%
  do({
    x <- .
    merge(
      x, 
      tibble(n=x$n[1],
        why=paste(
          "Amount: ", paste(format(round(as.numeric(x$amount)), nsmall=0, scientific=FALSE, big.mark=","), x$currency),
          "; Role: ", x$role,
          "; Dates: ", paste(x$start, "to", x$end),
          "; Proportion: ", x$perc),
        when=x$awarded
      )
    )
  })
```

Total income as PI, from `r subset(grant_totals, role=="PI")$n` grant(s): **`r subset(grant_totals, role=="PI")$incomep` GBP**

Total income as CI, from `r subset(grant_totals, role=="Co-I")$n` grant(s): **`r subset(grant_totals, role=="Co-I")$incomep` GBP**

Current income as PI, from `r subset(grant_totals_current, role=="PI")$n` grant(s): **`r subset(grant_totals_current, role=="PI")$incomep` GBP**

Current income as CI, from `r subset(grant_totals_current, role=="Co-I")$n` grant(s): **`r subset(grant_totals_current, role=="Co-I")$incomep` GBP**


```{r}
grantsf  %>%
  detailed_entries(with=title, why=why, when=when, what=funder)
```

## (iv) Indications of external recognition

#### Editorships

```{r}
tibble(
  with="PLoS Computational Biology",
  what="Invited guest editor",
  when="2017"
) %>%
  detailed_entries(what=what, when=when, with=with)
```

#### Appointment to national or international bodies

```{r}
tibble(
  with="Early Disease Detection Research Project",
  what="Member of chip design committee",
  when="2019",
  why="Invitation to a committee for designing the genotyping array for the Early Disease Detection Research Project, which will genotype 5 million UK participants by 2024"
) %>%
  detailed_entries(what=what, when=when, why=why, with=with)
```

#### Invitations for degree examinations

```{r}
read_csv(here::here("data", "degree_examination.csv")) %>%
  arrange(desc(when)) %>%
  detailed_entries(what=what, where=where, with=who, when=when)
```


#### Invited lectures (last 3 years)

```{r}
read_csv(here::here("data", "talks.csv")) %>% 
  arrange(desc(when)) %>%
  filter(when >= (now() %>% year() - 3)) %>%
  arrange(desc(when)) %>%
  detailed_entries(what=what, with=who, when=when, where=type, why=details)
```

## (vi) Related administration

```{r}
read_csv(here::here("data", "research_management.csv")) %>%
  mutate(where = as.character(where)) %>%
  arrange(desc(when)) %>%
  detailed_entries(what=with, where=where, with=what, when=when, why=why)
```

# Academic leadership and citizenship

## (i) Academic leadership in the discipline

I orchestrate the collection and harmonisation of data across hundreds of international research groups through the OpenGWAS project. This data infrastructure is the largest of its kind, and routinely gets more than 2 million queries per week from all over the world. 

I design analytical pipelines by unifying and developing statistical methods with researchers from multiple international groups. For example I developed the MR-Base software platform, which has now been cited nearly 1000 times since being published in 2018. I also developed the Genetics of DNA Methylation (GoDMC) analytical pipeline, which was recently diseminated to 36 groups around the world and resulted in a paper recently accepted in Nature Genetics. 

I form part of a group coordinating the world's largest genetic data collection effort. The Early Disease Detection Research Project (EDDRP) will genotype 5 million individuals in the UK within the next 4 years and I am part of the genotype chip design committee which meets on a monthly basis.

```{r}
academic_leadership <- read_csv(here::here("data", "academic_leadership.csv")) %>%
  mutate(where = as.character(where), why = as.character(why)) %>%
  arrange(desc(when)) %>%
  separate_why()

# academic_leadership %>%
#   filter(type == "international") %>%
#   arrange(desc(when)) %>%
#   detailed_entries(what=as.character(what), where=where, with=with, when=when, why=why)
```

## (ii) Academic leadership in the University

```{r}
academic_leadership %>%
  filter(type == "university") %>%
  arrange(desc(when)) %>%
  detailed_entries(what=as.character(what), where=where, with=with, when=when, why=why)
```

## (iii) Professional activities outside the University

```{r}
academic_misc <- read_csv(here::here("data", "academic_misc.csv")) %>%
  arrange(desc(when))

academic_misc %>%
  filter(type=="professional") %>%
  brief_entries(what=what, when=when)
```

## (iv) Contributions to society

```{r}
academic_misc %>%
  filter(type=="society") %>%
  brief_entries(what=what, when=when)
```

## (v) Entrepreneurship, enterprise and partnerships

```{r}
academic_misc %>%
  filter(type=="entrepreneurship") %>%
  brief_entries(what=what, when=when)
```

## (vi) Good citizenship

```{r}
academic_misc %>%
  filter(type=="citizenship") %>%
  brief_entries(what=what, when=when)
```

# Future plans

My fellowship is focused on implementing causal inference on a phenome-wide scale, creating a graph of the causal estimates of every phenotype against every other phenotype. I will create collaborations within the University and externally with experts in artificial intelligence, to explore new ways to exploit this graph for biological understanding and medical applications. I will use my causal graph to develop new ways to engage with the public, exploring how perceived ideas of medical interventions would shape future trajectories of population disease burden.

I plan to develop a new professional development course that guides junior researchers and post-graduate stu- dents through best practices in code and data management and reproducible digital research.

Through leading the special interest group on decolonising the curriculum, my goal is to develop a method of acreditation for courses across the medical school to introspectively examine potential biases in their teaching materials, identify ways of addressing them, and seek external review of proposed changes. This is a project of crucial importance for equiping the graduates and trainees in redressing biases that continue to incur health inequalities between ethnic groups.

